spec/unit/depgraph_spec.lua

```diff
@@ -98,4 +98,43 @@ describe("DepGraph module", function()
             'readerhighlight_hold_release',
         }, dg:serialize())
     end)
+
+    it("should serialize complex graph and keep dependancies after remove and re-add", function()
```

Comment:
I wouldn't have commented if that dependancies didn't stand out so much. :-P
```suggestion
    it("should serialize complex graph and keep dependencies after removing and re-adding", function()
```